setup.cfg

```diff
@@ -37,3 +37,6 @@ show_missing = True
 
 [tool:pytest]
 asyncio_mode = auto
+
+env =
+    PREFECT_LOGGING_TO_API_ENABLED = False
```

Comment:
Not necessary at this time — was a problem when testing streaming worker logs to the API